packages/swing-store/docs/swingstore.md

```diff
@@ -0,0 +1,52 @@
+# SwingStore Data Model
+
+The "SwingStore" provides a database to hold SwingSet kernel state, with an API crafted to help both the kernel and the host application mutate, commit, export, and import this state.
+
+The state is broken up into several pieces, or "stores":
+
+* `bundleStore`: a string-keyed Bundle-value table, holding source bundles which can be evaluated by `importBundle` to create vats, or new Compartments within a vat
+* `transcriptStore`: records a linear sequence of deliveries and syscalls (with results), collectively known as "transcript entires", for each vat
+* `snapStore`: records one or more XS heap snapshots for each vat, to rebuild a worker more efficiently than replaying all transcript entries from the beginning
+* `kvStore`: a string-keyed string-valued table, which holds everything else. Currently, this holds each vat's c-list and vatstore data, as well as the kernel-wide object and promise tables, and run-queues.
```

Comment:
Hm, I don't think so.. the `vatstore` is a database made available to all vats, via the four syscalls. That document explains how liveslots (as opposed to any other vat runtime) chooses to use the vatstore (which is an internal concern of liveslots, not something the swingstore should know anything about).

If we were to link it to anything, it should be to something that explains what the vatstore is good for.. some kind of "here's the environment that the kernel provides to all vats" document. I did a quick skim through https://github.com/Agoric/agoric-sdk/tree/master/packages/SwingSet/docs but we don't really have anything suitable there. https://github.com/Agoric/agoric-sdk/tree/master/packages/SwingSet/docs/vat-environment.md would be a candidate except that it focuses on the JS environment, rather than what's available on `syscall`.

Some day we might add a `syscalls.js` to that directory, at which point the section on the vatstore would be a good target. For now I'm going to leave it up to the reader to grep around for what a vatstore is.